website/docs/docs/cloud/dbt-cloud-ide/keyboard-shortcuts.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "dbt Cloud IDE keyboard shortcuts"
+description: "Learn about the keyboard shortcuts available in the Cloud IDE."
+sidebar_label: "Keyboard shortcuts"
+---
+
+
+The Cloud IDE provides keyboard shortcuts, features, and development tips to help you work faster and be more productive. 
+
+Use this dbt Cloud IDE  page to help you quickly reference some common operations.
+
+| Shortcut description | macOS | Windows |
+|--------|----------------|------------------|
+| View a full list of editor shortcuts | Fn-F1 | Fn-F1 |
+| Select a file to open | Command-O | Control-O |
+| Open the command palette | Command-P or Command-Shift-P | Control-P or Control-Shift-P |
+| Multi-edit by selecting multiple lines | Option-click or Shift-Option-Command | Hold Alt and click |
+| Preview code | Command-Enter | Control-Enter |
+| Compile code | Command-Shift-Enter | Control-Shift-Enter |
+| Preview or Compile selected code | (Use the above shortcuts) | (Use the above shortcuts) |
```

Comment:
I would consider deleting this line, as there are many cases where selected code works differently than just unselected code.